hub/router/proxy.go

```diff
@@ -27,6 +32,37 @@ var proxyTransport = &http.Transport{
 	IdleConnTimeout:     60 * time.Second,
 }
 
+// Get capability prefix from environment variable, default to "gads"
+var capabilityPrefix = getEnvOrDefault("GADS_CAPABILITY_PREFIX", "gads")
+
+func getEnvOrDefault(key, defaultValue string) string {
+	if value := os.Getenv(key); value != "" {
+		return value
+	}
+	return defaultValue
+}
+
+// extractGADSSecret extracts client secret from Appium session request
+func extractGADSSecret(sessionReq map[string]interface{}, prefix string) string {
+	// Check capabilities.alwaysMatch (W3C format)
+	if caps, ok := sessionReq["capabilities"].(map[string]interface{}); ok {
```

Comment:
We already have `CommonCapabilities` struct in `appiumgrid.go`. Maybe we can move it to `models` and reuse it here instead of parsing it like this?

Reply:
Done in [commit](https://github.com/shamanec/GADS/pull/217/commits/d49696e9ff095c44d699ceeed062302fdd952ad2)